dd-sdk-android-ndk/src/main/cpp/utils/backtrace-handler.cpp

```diff
@@ -41,51 +41,76 @@ namespace {
         return _URC_NO_REASON;
     }
 
-
     size_t capture_backtrace(uintptr_t *buffer, size_t max) {
         BacktraceState state = {buffer, buffer + max};
         // unwinds the backtrace and fills the buffer with stack lines addresses
         _Unwind_Backtrace(unwind_callback, &state);
         return state.current - buffer;
     }
 
-    const char *get_line_symbol(const void *addr) {
-        const char *symbol = "";
+    std::string address_to_hexa(uintptr_t address) {
+        char address_as_hexa[20];
+        #pragma clang diagnostic push
+        #pragma clang diagnostic ignored "-Wformat"
+        std::snprintf(address_as_hexa, sizeof(address_as_hexa), "0x%lx", address);
+        return std::string(address_as_hexa);
+        #pragma clang diagnostic pop
+    }
+
+    void getInfoFromAddress(std::string &backtrace, uintptr_t address) {
```

Comment:
FYI nit :-)

In my previous company style guide you should:
- use a pointer when an argument was "read/write", i.e. `std::string* backtrace`, rather than `std::string &backtrace` (the reference notation was only used for "read-only" parameters, to avoid making a copy, but then you needed to use `const`)
- have all the "read-only" parameters first, and then the "read/write" parameters, i.e. `void getInfoFromAddress(uintptr_t address, std::string* backtrace)` instead of what you have.